base_layer/wallet/src/output_manager_service/handle.rs

```diff
@@ -295,6 +315,24 @@ impl OutputManagerHandle {
         }
     }
 
+    pub async fn add_rewindable_output(
```

Comment:
Are `add_rewindable_output`,  `add_rewindable_output_with_tx_id`, `add_rewindable_unvalidated_output` only used for testing? I am assuming that they are because how would a normal client of this service be able to construct its own rewindable output externally.

So based on that I would say these three methods don't actually belong in API of this service. Now the question is how do you get these kind of outputs into the output manager for testing. The answer is that in the testing code you have access to the OMS database backend and you can insert them directly.

Reply:
The counter parts are used for importing of outputs manually and by the UTXO scanner.

Reply:
As discussed, I will remove these from the API.